ivy_tests/test_ivy/helpers/hypothesis_helpers/general_helpers.py

```diff
@@ -527,9 +527,7 @@ def embedding_helper(draw):
     dtype_weight, weight = draw(
         array_helpers.dtype_and_values(
             available_dtypes=[
-                x
-                for x in draw(dtype_helpers.get_dtypes("numeric"))
-                if "float" in x or "complex" in x
+                x for x in draw(dtype_helpers.get_dtypes("numeric")) if "float" in x
```

Comment:
Hello @baberabb, why we did we remove `complex` from the here? also if we only want `float` dtypes, we can use `draw(dtype_helpers.get_dtypes("float")` right away.

Reply:
Hi @CatB1t You're right, using `draw(dtype_helpers.get_dtypes("float"))` directly would have been better. I removed `complex` because embeddings typically only use real weights. Will change it back.